src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/FacebookSignupModalComponentObject.java

```diff
@@ -55,6 +62,29 @@ public void acceptWikiaAppPolicy() {
     }
   }
 
+  public void acceptWikiaAppPolicyNoEmail() {
+
+    Set<String> handles = driver.getWindowHandles();
+
+    if (handles.size() > 1) {
+      for (String winHandle : handles) {
+        //Switch to new window opened
+        driver.switchTo().window(winHandle);
+      }
+      waitForElementByElement(editInfoProvided);
+      editInfoProvided.click();
+      PageObjectLogging.log("acceptWikiaAppPolicyNoEmail", "editing info provided", true);
+      emailCheckbox.click();
```

Comment:
i'm getting element not clickable error at this point.
I don't see checkbox on the page when the test execution reaches this line, and it fails
We really don't want fresh tests to fail ! :)